### ISSUE TITLE
Training with ResNet

### ISSUE THREAD

**Issue by kfzyqin** [junsukchoe/ADL#14]
Hi, 

Can I please ask for the training details for training on the CUB dataset with ResNet? 

I tried both the pytorch and tensorflow implementations. For pytorch, I got 51% accuracy for top-1 localisation. For tensorflow, the accuracy is even worse, even I have used the SE-ResNet50. 

Can I please ask for the training details on how to get the localisation accuracy of 62.29% in your paper? For example, can you please provide the bash scripts that you have used? 

Thanks!

**Comment by junsukchoe** [comment]
Currently the PyTorch implementation is not stable, so I recommend you to use Tensorflow version for reproducing results.

You can use this script:

`python CAM-resnet.py --gpu 0 --data $YOUR_DATASET_PATH --cub --base-lr 0.1 --logdir ResNet50SE_CUB --load ResNet --stepscale 5.0 --batch 128 --depth 50 --mode se --attdrop 31 41 5 --drop_rate 0.75 --threshold 0.90`

If you have any further questions, please let me know.
